src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/bitbucket/AbstractPullrequest.java

```diff
@@ -23,6 +24,39 @@
         String getOwnerName();
 
         String getRepositoryName();
+
+        RepositoryLinks getLinks();
+    }
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class RepositoryLinks {
+        private List<RepositoryLink> clone = new ArrayList<>();
```

Comment:
Why is the list of RepositoryLink's called `clone`?

Seems confusing.

Also, add comments to classes, please. This is a simple one, so you don't need to comment about WHAT it's doing, but, I would like comments about what it is for, and where it is used.

Reply:
I've been lazy and just followed the JSON structure. I can rename to `cloneLinks` and put the JSON field name into an annotation.